docker/build.sh

```diff
@@ -79,10 +123,10 @@ EOF
   fi
 fi
 
-# Build FlexFlow Docker image
+# Build FlexFlow Docker image with input cuda version
 # shellcheck source=/dev/null
 . config/config.linux get-docker-configs
 # Set value of BUILD_CONFIGS
 get_build_configs
 
-docker build --build-arg "N_BUILD_CORES=${n_build_cores}" --build-arg "FF_GPU_BACKEND=${FF_GPU_BACKEND}" --build-arg "BUILD_CONFIGS=${BUILD_CONFIGS}" -t "flexflow-${FF_GPU_BACKEND}" -f docker/flexflow/Dockerfile .
+docker build --build-arg "N_BUILD_CORES=${n_build_cores}" --build-arg "FF_GPU_BACKEND=${FF_GPU_BACKEND}" --build-arg "BUILD_CONFIGS=${BUILD_CONFIGS}" -t "flexflow-${FF_GPU_BACKEND}-${cuda_version}" -f docker/flexflow/Dockerfile .
```

Comment:
because we changed the name of the `flexflow-environment` docker image to include the cuda version, now we will also need to change the Dockerfile for the flexflow image (`docker/flexflow/Dockerfile`) to use the correct `flexflow-environment` image as its base image. In other words, the FROM statement in the flexflow Dockerfile should also contain the proper cuda version suffix